dataMining/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "datamining",
+  "version": "1.0.0",
+  "description": "Data mining module of sup",
+  "main": "src/index.ts",
+  "scripts": {
+    "test": "jest",
+    "db:fixtures:load": "ts-node src/database/fixtures/load.ts",
+    "db:migration:create": "ts-node ./node_modules/.bin/typeorm migration:create",
+    "db:migration:generate": "ts-node ./node_modules/.bin/typeorm migration:generate",
```

Comment:
just a note, I am unsure about splitting migrations across modules... It could work, I just have no experience on this.